src/lib/qc.ml

```diff
@@ -135,7 +135,69 @@ module EDSL = struct
       to_email: string;
       mailgun_api_key: string;
       mailgun_domain_name: string; }
-    [@@deriving show,make]
+  [@@deriving show,make]
```

Comment:
Because it's an option type, and I don't see a nice way to handle that here (unless I made all the options within the QC type optional, which isn't really what I want here.) 

I wouldn't mind doing some work on Ppx_deriving_cmdliner to support a case like this, but I'm not sure a good general way to handle it.